sdks/python/setup.py

```diff
@@ -159,7 +160,7 @@ def run(self):
     author_email=PACKAGE_EMAIL,
     packages=setuptools.find_packages(),
     package_data={'apache_beam': [
-        '*/*.pyx', '*/*/*.pyx', '*/*.pxd', '*/*/*.pxd', 'testing/data/*']},
+        '*/*.pyx', '*/*/*.pyx', '*/*.pxd', '*/*/*.pxd', 'testing/data/*.yaml']},
```

Comment:
<!--new_thread; commit:04316420b4d4ecc4bc33561e8abb33a183bc9937; resolved:0-->
Why do we need *.yaml files ?

Reply:
We want to include the *.yaml files for other tests, but exclude everything in the vcf directory since these files are very large.